docs/configuring-playbook-backup-borg.md

```diff
@@ -10,18 +10,21 @@ The backup will run based on `backup_borg_schedule` var (systemd timer calendar)
 
 By default, if you're using the integrated Postgres database server (as opposed to [an external Postgres server](configuring-playbook-external-postgres.md)), backups with BorgBackup will also include dumps of your Postgres database. An alternative solution for backing up the Postgres database is [postgres backup](configuring-playbook-postgres-backup.md). If you decide to go with another solution, you can disable Postgres-backup support for BorgBackup using the `backup_borg_postgresql_enabled` variable.
 
+**Note**: the `matrix-backup-borg` role is not managed by this repository but its [own repository](https://github.com/mother-of-all-self-hosting/ansible-role-backup_borg).
```

Comment:
Why call it a `matrix-backup-borg` role?

Maybe it was historically called that way (before becoming independent), but doesn't mentioning that old name here confuse people?

Maybe you meant to say "component" or "service" instead of "role" in this sentence?